plugin/evm/vm.go

```diff
@@ -1308,6 +1385,70 @@ func (vm *VM) GetSpendableAVAXWithFee(
 	return inputs, signers, nil
 }
 
+// getSpendableAVAXWithFeeEVMInputForAddress returns an EVMInput
+//  to total [amount] + [fee] of [AVAX] owned by [address]. Additionally,
+// it returns a new [cost] and [amount] for use by the caller if this method is
+// run in a loop. This function accounts for the added cost of the additional
+// inputs needed to create the transaction and makes sure to skip any keys with a
+// balance that is insufficient to cover the additional fee.
+func (vm *VM) getSpendableAVAXWithFeeEVMInputForAddress(
```

Comment:
Refactor contents of `GetSpendableAVAXWithFee` loop so that this method can accept a `common.Address` instead of a `*crypto.PrivateKeySECP256K1R`